common/types/supportedFileTypes.ts

```diff
@@ -5,6 +5,11 @@ export enum SupportedRawFileTypes {
   TXT = "TXT",
 }
 
+export enum InputKeyTypes {
+  INPUT = "input",
+  SELECT = "select",
+}
+
```

Comment:
Define this in the filter-block component. (It's not a common data structure because it's only used in the frontend with the filter-block component)

Reply:
done